app/experimenter/nimbus-ui/src/components/PageResults/TableMetricCount/index.tsx

```diff
@@ -34,6 +34,7 @@ type TableMetricCountProps = {
   outcomeDefaultName: string;
   group: string;
   metricType?: MetricTypes;
+  segment?: string;
```

Comment:
In TypeScript there's this awesome thing where you can declare a value as an enum of string constants and then later even if you refer to something as a literal value it'll still validate that it's one of the valid enum choices.  If we know the segment keys here we should declare them explicitly as like an enum or a string union, and then even if you use the literal string value anywhere in the code it'll still know whether it's a valid choice.

Reply:
Oh you know I just realized, segments will be arbitrary keys that come from the data, so we can't do what I just said.  Oh well.

Reply:
Yea... I had the exact same thought process and decided that `string` was the best I could do.